libraries/testing/include/eosio/testing/tester.hpp

```diff
@@ -895,15 +895,15 @@ namespace eosio::testing {
       // different nodes should use different keys
       // OK to configure keys not used in a finalizer_policy
       // -------------------------------------------------------------
-      void set_node_finalizers(size_t first_key, size_t num_keys) {
-         node_first_key = first_key;
+      void set_node_finalizers(size_t first_key_index, size_t num_keys) {
+         node_first_key_idx = first_key_index;
          node_num_keys = num_keys;
-         t.set_node_finalizers({&key_names.at(first_key), num_keys});
+         t.set_node_finalizers({&key_names.at(first_key_index), num_keys});
       }
 
       void set_node_finalizers() {
          if (node_num_keys)
-            t.set_node_finalizers({&key_names.at(node_first_key), node_num_keys});
+            t.set_node_finalizers({&key_names.at(node_first_key_idx), node_num_keys});
       }
 
       // updates the finalizer_policy to the `fin_policy_size` keys starting at `first_key`
```

Comment:
The comment about `first_key` should be changed too. But OK not to touch them this time.